README.md

```diff
@@ -44,6 +44,35 @@ a token provided by the web app.
 As a starting point please check ```ApiModule``` class - it provides an
 entry point you need to know to communicate with the web application. 
 
+### Troubleshooting
+
+#### Gradle synchronization failure
+
+If you encounter this error:
```

Comment:
It's in petunia project because it is related to that module (petunia module is the actual app). No need for it to be global.